pydrive2/fs/spec.py

```diff
@@ -376,9 +376,13 @@ def gdrive_get_file(self, item_id, rpath, callback):
 
         extra_args = {}
         if callback:
+
+            def cb(inc, _):
+                callback.relative_update(inc)
```

Comment:
I am not sure if this should be relative_update or absolute_update. I have been trying to run the tests locally, but haven't been able to yet.

Reply:
It was `absolute_size`.